uint8 desired_blob              # Desired Blob index. Default is 0 
float64 aspect_ratio            # Aspect ratio of the color blob  
float64 aspect_ratio_threshold  # Threshold +/- the given aspect ratio to still look for object
float64 size                    # expected blob length size to filter out similar aspect ratio noise blobs
float64 size_threshold          # Threshold +/- the given size to still look for object
string color                    # color of the mask to use
string prefix                   # prefix for image topics names
---
geometry_msgs/PoseStamped centroid_pose 
sensor_msgs/Image color_img
sensor_msgs/Image color_img_raw
sensor_msgs/Image mask
sensor_msgs/Image depth_img
sensor_msgs/CameraInfo cam_info